src/test/java/uk/gov/pay/ledger/transaction/model/CsvTransactionFactoryTest.java

```diff
@@ -71,40 +73,95 @@ public void toMapShouldReturnMapWithCorrectCsvDataForPaymentTransaction() {
     }
 
     @Test
-    public void toMapShouldReturnMapWithCorrectCsvDataForRefundTransaction() {
+    void toMapShouldReturnMapWithCorrectCsvDataForRefundTransaction() {
 
-        TransactionFixture refundTransactionFixture = aTransactionFixture()
-                .withCreatedDate(ZonedDateTime.parse("2018-03-12T16:25:01.123456Z"))
-                .withTransactionType("REFUND")
-                .withAmount(99L)
-                .withTotalAmount(99L)
+        var transactionEntity = transactionFixture
+                .withTransactionType(TransactionType.REFUND.name())
                 .withState(TransactionState.ERROR)
+                .withTotalAmount(123L)
+                .withRefundedByUserEmail("refundedbyuser@example.org")
                 .withGatewayTransactionId("refund-gateway-transaction-id")
                 .withRefundedByUserEmail("refundedbyuser@example.org")
                 .withParentExternalId("parent-external-id")
+                .withCorporateCardSurcharge(0)
                 .withDefaultPaymentDetails()
-                .withDefaultTransactionDetails();
+                .withDefaultTransactionDetails()
+                .toEntity();
 
-        Map<String, Object> csvDataMap = csvTransactionFactory.toMap(refundTransactionFixture.toEntity());
+        Map<String, Object> csvDataMap = csvTransactionFactory.toMap(transactionEntity);
 
-        assertPaymentDetails(csvDataMap, refundTransactionFixture.toEntity());
-        assertThat(csvDataMap.get("Amount"), is("-0.99"));
-        assertThat(csvDataMap.get("Provider ID"), is(refundTransactionFixture.getGatewayTransactionId()));
-        assertThat(csvDataMap.get("GOV.UK Payment ID"), is(refundTransactionFixture.getParentExternalId()));
+        assertPaymentDetails(csvDataMap, transactionEntity);
+        assertThat(csvDataMap.get("Amount"), is("-1.00"));
```

Comment:
Is this default for the amount set somewhere ? `Total Amount` is `-1.23` and the amount is `-1.00` but for refund these should be the same. Guessing this is because of test setup ?

Reply:
Yes, the fixture in the setup method sets `amount` to `100L`. I'll clarify the value in the test case.